v2/internal/controllers/samples_test.go

```diff
@@ -39,6 +39,7 @@ var randomNameExclusions = []string{
 	"compute",
 	"documentdb",
 	"insights",
+	"kubernetesconfiguration",
```

Comment:
minor: Why does this need to avoid random name exclusions?

Reply:
Cz the owner exists as a reference object, if we use the random names, there's no way the sample can find the owner if we use random names.

Reply:
Now if I think more of it.. its a tech-debt - I'll open an issue for this and add a todo on the exclusion.

Reply:
Missing a TODO here?

Reply:
Done!